src/azure/Templates/AzureServiceClientTemplate.cshtml

```diff
@@ -77,7 +77,7 @@ else
             self@(Model.RequiredConstructorParameters)@(Model.IsCustomBaseUri ? "" : ", base_url=None"), config=None, **kwargs):
 @EmptyLine
         self._config = config or @(Model.Name)Configuration(@(Model.ConfigConstructorParameters)**kwargs)
-        self._client = PipelineClient(base_url=@(!Model.IsCustomBaseUri? "base_url": PythonConstants.None), credentials=@(Settings.AddCredentials? "credentials" : PythonConstants.None), config=self._config, **kwargs)
+        self._client = PipelineClient(base_url=@(!Model.IsCustomBaseUri? "base_url": PythonConstants.None), config=self._config, **kwargs)
```

Comment:
I made some modifications here - though I'm not sure why the diff is comparing to the latest code in azure-core....
https://github.com/Azure/autorest.python/commit/5ddcccea02fd136717619df94519fdbe536ce9b2